modules/social.py

```diff
@@ -1043,6 +1098,71 @@ async def _profile_validate_allow(self, ctx, member: tools.ResolveUser):
         self.validate_allowed_users.append(member.id)
         return await ctx.message.reply(f'{config.greenTick} {member} temporarily added to allowlist')
 
+    @commands.has_any_role(config.moderator, config.eh)
+    @commands.group(name='trivia')
+    async def _trivia(self, ctx):
+        return
+
+    @commands.has_any_role(config.moderator, config.eh)
+    @_trivia.command(name='award')
+    async def _trivia_award(self, ctx, members: commands.Greedy[tools.ResolveUser]):
+        '''Increase the trivia award trophy by one tier for one or more users'''
+        stats = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0, 7: 0, 8: 0, 9: 0}
```

Comment:
Again, if you had used an array, this could have been:
`stats = [0] * 10`
or even better:
`stats = [0] * len(self.triviaTrophyIndex)`